content/20.pharmaceuticals.md

```diff
@@ -699,52 +699,88 @@ Additionally, different studies used different dosages, number of doses, and met
+The interim analysis of the phase 2 portion suggested that bamlanivimab alone was able to reduce accelerate the reduction in viral load [@doi:10.1056/NEJMoa2029849].
+However, more recent data suggests that only the bamlanivimab/etesevimab combination therapy is able to reduce viral load in COVID-19 patients [@doi:10.1001/jama.2021.0202]. 
+Based on this data, the combination therapy received emergency use authorization for COVID-19 from the FDA in February of 2021.
+
+##### Casirivimab and Imdevimab (REGN-COV2) 
+
+Casirivimab (REGN10933) and imdevimab (REGN10987) are two monoclonal antibodies against the SARS-CoV-2 spike protein.
+They were both developed by Regeneron in a parallel high-throughput screen to identify neutralizing antibodies from either humanized mice or patient-derived convalescent plasma [@doi:10.1126/science.abd0827].
+In these efforts, multiple antibodies were characterized for their ability to bind and neutralize the SARS-CoV-2 spike protein.
+The authors hypothesized that an antibody cocktail, rather than each individual antibody could increase the therapeutic efficacy while minimizing the risk for virus escape.
+Therefore, the authors tested pairs of individual antibodies for their ability to simultaneously bind the RBD of the spike protein.
+Based on this data, casirivimab and imdevimab were identified as the lead antibody pair, resulting in the initiation of two clinical trials [@clinicaltrials:NCT04426695; @clinicaltrials:NCT04425629].
+Data from this phase 1-3 trial published in the _New England Journal of Medicine_ shows that the REGN-COV2 antibody cocktail reduced viral load, particularly in patients with high viral load or whose endogenous immune response had not yet been initiated [@doi:10.1056/NEJMoa2035002].
+However, in patients who already initiated an immune response, exogenous addition of REGN-COV2 did not improve the endogenous immune response.
+Both doses were well tolerated with no serious events related to the antibody cocktail.
+Based on this data, the FDA granted emergency use authorization for REGN-COV2 in patients with mild to moderate COVID-19 who are at risk to develop severe disease.
+Ongoing efforts are trying to evaluate the efficacy of REGN-COV2 to improve clinical outcomes in hospitalized patients [@clinicaltrials:NCT04426695].
+
+##### Viral Resistance to Neutralizing Antibodies
+
+With the ongoing global spread of new SARS-CoV-2 variants, there is a growing concern that mutations in SARS-CoV-2 spike protein could escape antibody neutralization, thereby reducing the efficacy of monoclonal antibody therapeutics and vaccines.
+A comprehensive mutagenesis screen recently identified several amino acid substitutions in the SARS-CoV-2 spike protein that can prevent antibody neutralization [@doi:10.1016/j.chom.2021.01.014].
```

Comment:
is 'escape from neutralization' the same as 'escape from antibody recognition (i.e. antibody binding)'?

Reply:
It's one of these situations: You cant have neutralization without binding but you can have binding without neutralization. Antibodies themselves bind the target with their variable region. The constant region of an antibodies isotype is subsequently recognized by the innate immune system that ultimately clears the infection. There are approaches to enhance the efficacy of antibodies by modulating the constant region of an antibody to more effectively engage other immune cells.